src/pages/workspaces/List.js

```diff
@@ -103,7 +103,7 @@ const somePersistentDiskRequiresMigration = (workspaces, disks) => {
     _.mapValues(_.flow(_.groupBy('name'), _.mapValues(_.head)))
   )(workspaces)
 
-  return workspaces && _.flip(_.some)(disks, ({ googleProject, labels }) => {
+  return !_.isEmpty(workspaces) && _.flip(_.some)(disks, ({ googleProject, labels }) => {
```

Comment:
This avoids the message flashing while we are in the state of `workspaces` being an empty list (which is truthy) and the `disks` variable being populated.

Reply:
TIL - empty list is truthy. because of course it is.